members/F000062.yaml

```diff
@@ -1,71 +1,105 @@
+  action: ""
   steps:
-    - visit: https://www.feinstein.senate.gov/public/index.cfm/e-mail-me
+    - visit: "https://www.feinstein.senate.gov/public/index.cfm/e-mail-me"
     - fill_in:
-      - name: field_0c5a1392-6512-4fa5-a213-eb7e1cbaf1ad
-        selector: "#field_0c5a1392-6512-4fa5-a213-eb7e1cbaf1ad"
-        value: "$NAME_FIRST"
-        required: Yes
-      - name: field_ac791fc0-5e57-4f5e-88d4-6ef9e730f470
-        selector: "#field_ac791fc0-5e57-4f5e-88d4-6ef9e730f470"
-        value: "$NAME_LAST"
-        required: Yes
-      - name: field_844e2e93-7496-4372-bb04-d0cf157684f0
-        selector: "#field_844e2e93-7496-4372-bb04-d0cf157684f0"
-        value: "$ADDRESS_STREET"
-        required: Yes
-      - name: field_289b92dd-13e8-40e7-b141-1a02c6acdcea
-        selector: "#field_289b92dd-13e8-40e7-b141-1a02c6acdcea"
-        value: "$ADDRESS_STREET_2"
-      - name: field_c44b713c-04ad-411a-9744-6e7f2ff71898
-        selector: "#field_c44b713c-04ad-411a-9744-6e7f2ff71898"
-        value: "$ADDRESS_CITY"
-        required: Yes
-      - name: field_64d2124b-a227-4f90-ad83-65dbc8175a69
-        selector: "#field_64d2124b-a227-4f90-ad83-65dbc8175a69"
-        value: "$ADDRESS_ZIP5"
-        required: Yes
-      - name: field_d065bdd9-7c1e-4d99-a50e-6a1adae8803b
-        selector: "#field_d065bdd9-7c1e-4d99-a50e-6a1adae8803b"
-        value: "$EMAIL"
-        required: Yes
-      - name: field_130638f0-2eec-4c6b-83be-11e83bb5ced3
-        selector: "#field_130638f0-2eec-4c6b-83be-11e83bb5ced3"
-        value: "$MESSAGE"
-        required: Yes
+        - name: firstname
+          selector: "#field_0C5A1392-6512-4FA5-A213-EB7E1CBAF1AD"
+          value: $NAME_FIRST
+          required: true
+        - name: lastname
+          selector: "#field_AC791FC0-5E57-4F5E-88D4-6EF9E730F470"
+          value: $NAME_LAST
+          required: true
+        - name: street
+          selector: "#field_844E2E93-7496-4372-BB04-D0CF157684F0"
+          value: $ADDRESS_STREET
+          required: true
+        - name: street2
+          selector: "#field_289B92DD-13E8-40E7-B141-1A02C6ACDCEA"
+          value: $ADDRESS_STREET_2
+          required: false
+        - name: city
+          selector: "#field_C44B713C-04AD-411A-9744-6E7F2FF71898"
+          value: $ADDRESS_CITY
+          required: true
+        - name: zip5
+          selector: "#field_64D2124B-A227-4F90-AD83-65DBC8175A69"
+          value: $ADDRESS_ZIP5
+          required: true
+        - name: email
+          selector: "#field_D065BDD9-7C1E-4D99-A50E-6A1ADAE8803B"
+          value: $EMAIL
+          required: true
+        - name: message
+          selector: "#field_130638F0-2EEC-4C6B-83BE-11E83BB5CED3"
+          value: $MESSAGE
+          required: true
+    - choose:
+        - name: topic
+          selector: "#field_302E8A41-000D-419E-991E-40C7CB96F97C_14"
+          value: Government Affairs
+          required: true
+          options:
```

Comment:
Go ahead and just leave the options off this step since it isn't part of the spec. We're going to need a new step type to handle options later.